packages/apputils-extension/src/index.ts

```diff
@@ -581,6 +583,20 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
         }
       }
     });
+
+    commands.addCommand(CommandIDs.runAllCommands, {
+      label: trans.__('Run All Commands Passed as Args'),
```

Comment:
Maybe just "Run All Commands" or "Run Specified Commands"?

Reply:
How about RunAllEnabled for consistency? With a default of silently skipping disabled commands, but you can give an arg that would cause us to throw an error if a command was not enabled when we tried to run it?